test/tests/divertor_monoblock/elm_transient_sobol.i

```diff
@@ -0,0 +1,160 @@
+[StochasticTools] # Designate as the Controller/Main Input
+[]
+[MultiApps] # Designate a subapp to control later
+  [runner]
+    type = SamplerFullSolveMultiApp
+    sampler = sobol
+    input_files = 'elm_transient_runner.i'
+    mode = batch-reset
+    keep_full_output_history = True
+  []
+[]
+[Controls] # Control inputs from Main->Subapp
+  [cmdline]
+    type = MultiAppSamplerControl
+    multi_app = runner
+    sampler = sobol
+    param_names = """elm_value
+                     elm_duration
+                     coolant_temp
+                     W_cond_factor"""
+  []
+[]
+[Distributions] # Define probability distributions of parameters for sampling
+  [P_val]
+    type = Normal
+    mean =  1147e6
+    standard_deviation = 57.35e6
+  []
+  [P_dur]
+    type = Normal
+    mean =  1.32e-3
+    standard_deviation = 0.066e-3
+  []
+  [C_tem]
+    type = Normal
+    mean =  552
+    standard_deviation = 27.6
+  []
+  [W_cond]
+    type = Uniform
+    lower_bound = 0.9
+    upper_bound = 1.0
+  []
+[]
+[Samplers] # Sampling methodology using the probability distributions above
+  [hypercube_1]
+    type = LatinHypercube
+    distributions = 'P_val P_dur C_tem W_cond'
+    num_rows = 10 # N Samples
+    seed = 1001
+    execute_on = 'PRE_MULTIAPP_SETUP'
+  []
+  [hypercube_2]
+    type = LatinHypercube
+    distributions = 'P_val P_dur C_tem W_cond'
+    num_rows = 10 # N Samples
+    seed = 1002
```

Comment:
i would recommend defining these numbers at the top of the input file with units.